packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -28,6 +28,8 @@ export interface DescriptionListProps extends Omit<React.HTMLProps<HTMLDListElem
   isCompact?: boolean;
   /** Sets a horizontal description list to have fluid styling. */
   isFluid?: boolean;
+  /** Sets the description list to fill the entire column space. */
+  isFillColumns?: boolean;
```

Comment:
Maybe we can say `Sets the the default placement of description list groups to fill from top to bottom`, so the description aligns more with what core notes in their example.